### DISCUSSION TITLE
Getting User ID

### DISCUSSION THREAD

**Discussion by jelolul** [GodotNuts/GodotFirebase#223] [Q&A] [answered]
Hey! How would I get the user ID when the user logs in? I have been looking everywhere but couldn't find anything.

Great plugin by the way! 👍

**Comment by BearDooks** [comment, marked as answer]
@JeloLeaks I apologize I didn't see this sooner. You should be able to run this function after a user is logged in to pull all of their data

https://github.com/GodotNuts/GodotFirebase/wiki/Authentication-and-User-Management#get-user-data

I also tend to print out the user data to the console on a successful login

Chuck

**Comment by WolfgangSenff** [comment]
The default auth variable that returns should also contain their ID. You can just use that when authentication is successful. (Feel free to print that variable to see what all is in it.)
